hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/SPUIComponentProvider.java

```diff
@@ -199,7 +198,7 @@ public static Panel getDistributionSetInfo(final DistributionSet distributionSet
      * @return Label
      */
     public static Label createNameValueLabel(final String label, final String... values) {
-        final String valueStr = StringUtils.join(Arrays.asList(values), " ");
+        final String valueStr = StringUtils.arrayToCommaDelimitedString(values);
```

Comment:
Now the String is comma separated. The original was separated by a space character.